math-doc/math/scribblings/math-special-functions.scrbl

```diff
@@ -76,16 +76,17 @@ When @racket[x] is an exact integer, @racket[(gamma x)] is exact.
                  (gamma -1.0)
                  (gamma 0.0)
                  (gamma -0.0)
+                 (gamma 1+i)
                  (gamma 172.0)
                  (eval:alts
                   (bf (gamma 172))
                   (eval:result @racketresultfont{(bf "1.241018070217667823424840524103103992618e309")}))]
 
-Error is no more than 10 @tech{ulps} everywhere that has been tested, and is usually no more than 4
-ulps.
+On the real line the error is no more than 10 @tech{ulps} everywhere that has been tested, and is usually no more than 4
+ulps. In the rest of the complex plane the relative error is smaller than 1e-13 (@tech{ulps}=450).
```

Comment:
I'm confused by this comment. Does the parenthetical mean "smaller than 450 ulps"?

Reply:
I don't remember why I wrote that. I think I should just drop the part about ulps.
Also I did some extra testing and in the very close region around negative integers the error will be bigger.
I need to think on how to write this down

Reply:
It may be better to leave a vaguer "Outside the real line, errors are significantly higher". If we're not certain of the guarantees we shouldn't give them, and should warn users that they are higher.